terraform-modules/aws/amplify/default_iam_policy.json

```diff
@@ -0,0 +1,121 @@
+{
```

Comment:
Can all of the 
```
            "Resource": "*"
```
be tightened up to be more specific?  Im pretty sure having a `*` here will sound off alarm bells on multiple IAM policy scanners that we have and are using.

Reply:
updated